.github/actions/abort-release-candidate-v1/README.md

```diff
@@ -0,0 +1,39 @@
+# abort-release-candidate-v1
+
+A GitHub Action to abort a release candidate.
+
+## Inputs
+
+| Name        | Required | Description                                                                              | Default |
+| ----------- | -------- | ---------------------------------------------------------------------------------------- | ------- |
+| `token`     | Yes      | A GitHub token used for octokit and GH CLI with the [required permissions](#permissions) | NA      |
+| `base`      | Yes      | The base branch the release candidate was going to be merged into                        | NA      |
+| `docs-repo` | No       | The docs repo where the release notes issue is located                                   | NA      |
```

Comment:
This should be `repo` and `owner` to it can be used correctly in `get-release-issue`

Reply:
Do we want it to be that vague? The abort-release-candidate already attempts to get the issue URL for the RC: https://github.com/dequelabs/axe-api-team-public/pull/102/files#diff-58d5337330f7e542de2ed43b7f979a91c8ea07477f1397b364e19e9816d5ff48R40 - this input is explicitly for looking issues in the docs-repo for a given RC.

Reply:
can you easily split out the single `docs-repo` into `repo` and `owner`?

Reply:
I think for all cases where we intend to use `docs-repo` we can assume the `owner` will always be `dequelabs` so there is no need to supply that input. If we think that might change, we could add `docs-owner` and `docs-repo` and not just `owner` and `repo, thoughts?

Reply:
Good point. I withdraw my feedback.